webapp/src/app.tsx

```diff
@@ -4726,6 +4726,12 @@ pxt.winrt.captureInitialActivation();
 document.addEventListener("DOMContentLoaded", () => {
     pxt.perf.recordMilestone(`DOM loaded`)
 
+    // Initialization order is important here.
```

Comment:
Curious what people think of this change.

Reply:
It seems reasonable to me; this whole event handler is already pretty order dependent anyways, with all the initializations & the promise chain at the end, so keeping these important inits in one place seems good

Reply:
LGTM. Not sure it should impact much of anything to initialize the shell early. Could maybe move loadcache outside of the event, since it would have been loaded on module load before.

Reply:
oh wait, i guess you can't move it out unless you also move out the init. in any case, shouldn't matter much!

Reply:
Or I could move into pxtlib. It's pretty small. Let me try that.

Reply:
shell. move shell :)